tests/checkup_test.go

```diff
@@ -134,6 +138,32 @@ func setupCheckupPermissions() {
 		)
 		Expect(err).NotTo(HaveOccurred())
 	})
+
+	kubeVirtRTCheckerRole = newKubeVirtRTCheckerRole()
+	_, err = client.RbacV1().Roles(testNamespace).Create(context.Background(), kubeVirtRTCheckerRole, metav1.CreateOptions{})
+	Expect(err).NotTo(HaveOccurred())
+
+	DeferCleanup(func() {
+		err = client.RbacV1().Roles(testNamespace).Delete(context.Background(), testKubeVirtRTCheckerRoleName, metav1.DeleteOptions{})
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	kubeVirtRTCheckerRoleBinding = newKubeVirtRTCheckerRoleBinding()
+	_, err = client.RbacV1().RoleBindings(testNamespace).Create(
+		context.Background(),
+		kubeVirtRTCheckerRoleBinding,
+		metav1.CreateOptions{},
+	)
+	Expect(err).NotTo(HaveOccurred())
+
+	DeferCleanup(func() {
+		err = client.RbacV1().RoleBindings(testNamespace).Delete(
+			context.Background(),
+			testKubeVirtRTCheckerRoleBindingName,
```

Comment:
It would be cleaner to take the name from `kubeVirtRTCheckerRoleBinding` object itself.

Reply:
Done.